README.md

```diff
@@ -4,3 +4,10 @@ This repo contains the STEP capstone project: "Voter Central".
 
 * Live server: [go/voter-central](http://go/voter-central)
 * Design doc: [go/voter-central-dd](http://go/voter-central-dd)
+
+The project requires the use of AngularJS. To install, make sure you have
+a recent version of `npm` installed, then run this command:
+
+```bash
+npm install angular
+```
```

Comment:
Maybe move this instruction to the /project directory and say something like "Under `src/main/webapp/` run `npm install angular`".  Discovered the problems I was having with deploying this locally was that the webapp doesn't really "see" anything under the `webapp` directory.  Once that's done, you don't have to use relative paths for importing angular.  Can just use `/node_modules/angular/angular.js` in the html files.